pkg/kubelet/container/runtime_cache.go

```diff
@@ -94,3 +94,25 @@ func (r *runtimeCache) getPodsWithTimestamp() ([]*Pod, time.Time, error) {
 	pods, err := r.getter.GetPods(false)
 	return pods, timestamp, err
 }
+
+// TestRunTimeCache embeds runtimeCache with some additional methods for testing.
```

Comment:
did the simple rename of the original file to a non-test file, and added a comment